codegen/sdk-codegen/aws-models/ecs.2014-11-13.json

```diff
@@ -3080,6 +3080,51 @@
                             }
                         ],
                         "minDelay": 15
+                    },
+                    "ServicesStable": {
+                        "acceptors": [
+                            {
+                                "state": "failure",
+                                "matcher": {
+                                    "output": {
+                                        "path": "failures[].reason",
+                                        "expected": "MISSING",
+                                        "comparator": "anyStringEquals"
+                                    }
+                                }
+                            },
+                            {
+                                "state": "failure",
+                                "matcher": {
+                                    "output": {
+                                        "path": "services[].status",
+                                        "expected": "DRAINING",
+                                        "comparator": "anyStringEquals"
+                                    }
+                                }
+                            },
+                            {
+                                "state": "failure",
+                                "matcher": {
+                                    "output": {
+                                        "path": "services[].status",
+                                        "expected": "INACTIVE",
+                                        "comparator": "anyStringEquals"
+                                    }
+                                }
+                            },
+                            {
+                                "state": "success",
+                                "matcher": {
+                                    "output": {
+                                        "path": "length(services[?!(length(deployments) == `1` && runningCount == desiredCount)]) == `0`",
```

Comment:
After updating to smithy-swift 1.22.0, this is the line that is making the tests fail